examples/rotation/main.py

```diff
@@ -1,72 +1,62 @@
 import os
 import math
 from kivy.app import App
-from kivy.clock import Clock
-from kivy.graphics.stencil_instructions import StencilPush, StencilPop
 from kivy.properties import ObjectProperty
 from kivy.uix.boxlayout import BoxLayout
-from kivy.uix.button import Button
-
-from kivy3 import (
-    Scene,
-    Renderer,
-    PerspectiveCamera,
-    Geometry,
-    Vector3,
-    Material,
-    Mesh,
-    Face3,
-)
+from kivy3 import Scene
+from kivy3 import Renderer
+from kivy3 import PerspectiveCamera
+from kivy3 import Geometry
+from kivy3 import Material
+from kivy3 import Mesh
```

Comment:
I'm still not convinced it's the most Pythonic way to deal with this :sweat_smile: 
For instance look how `black` or `isort` are dealing with imports